Source/WebCore/platform/graphics/cocoa/SourceBufferParser.h

```diff
@@ -80,7 +80,7 @@ class WEBCORE_EXPORT SourceBufferParser : public ThreadSafeRefCounted<SourceBuff
         enum class ReadError { EndOfFile, FatalError };
         using ReadResult = Expected<size_t, ReadError>;
 
-        ReadResult read(size_t position, size_t, uint8_t* destination) const;
+        ReadResult read(size_t position, std::span<uint8_t> destination) const;
```

Comment:
This puts `position` first, but `FragmentedSharedBuffer::copyTo` puts `offset` second. A little messy.